contract_test/test_data_json_contract.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+import requests
+from jsonschema import validate
+
+
+def _get_instance():
+    return requests.get("https://audiobooks.by/data.json").json()
+
+
+def _get_schema():
+    with open("audiobooks.schema.json", "r") as f:
+        return json.load(f)
+
+
+def test_mytest():
```

Comment:
It would be good to run this test as part of our regular website tests on the fake data. So that when someone updates data.json generation - they are aware that they need to update schema as well (and potentially have some process around schema update).

Reply:
done. 
I've added such check to the existed data.json test. Additionally created separated test file for production data.json